templates/terraform_setup.yml

```diff
@@ -3,58 +3,140 @@
 parameters:
 - name: original_node_count
   type: string
-- name: run_terraform
+- name: tf_config
   type: string
 - name: image
   type: string
   default: "microsoft-avere:vfxt:avere-vfxt-node:latest"
   values:
     - "microsoft-avere:vfxt:avere-vfxt-node:latest"
+    - "microsoft-avere:vfxt:avere-vfxt-node:8.0.1"
+    - "microsoft-avere:vfxt:avere-vfxt-node:6.3.0"
+    - "microsoft-avere:vfxt:avere-vfxt-node:6.2.0"
     - "microsoft-avere:vfxt:avere-vfxt-node:6.0.0"
     - "microsoft-avere:vfxt:avere-vfxt-node:5.3.61"
```

Comment:
I'd suggest removing all but 6.3.0 and 8.0.1 (and leaving both `latest` entries). The 6.x.x line really should only be 6.3.0 at this point, and the 5.3.x line is quite old.

Reply:
got it!